shuup/front/apps/personal_order_history/templates/shuup/personal_order_history/order_detail.jinja

```diff
@@ -15,7 +15,8 @@
 {% endblock %}
 
 {% block dashboard_content %}
-    <div class="row">
+    <h1 class="page-header">Order #{{ order.id }}{{ render_action_buttons(order) }}</h1>
+    <div class="row order-dashboard-container">
```

Comment:
h1 is required because the header won't work (buttons won't align) properly without that. The solution is also in line with other views, we can unify this later imo.